cmd/notifier/messenger.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"log"
+	"time"
+
+	"github.com/status-im/status-go/geth/node"
+	"github.com/status-im/status-go/messenger"
+)
+
+const (
+	// defaultWorkTime is a work time reported in messages sent to MailServer nodes.
+	defaultWorkTime = 5
+)
+
+var (
+	// ErrInvalidSymKeyID is returned when it fails to get a symmetric key.
+	ErrInvalidSymKeyID = errors.New("invalid symKeyID value")
+)
+
+type BroadcastMsg struct {
+	PubKey string `json:"pubkey"`
+}
+
+// RegistrationRequestMsg : peers wanting to use this notification server will
+// send anonymous whisper messages with the device registration token, and
+// a secure channel, so communication between PN server and client can happen
+// securely
+type RegistrationRequestMsg struct {
+	DeviceRegistrationToken string `json:"token"`
+	SecureChannel           []byte `json:"channel"`
```

Comment:
The channel is composed of the symkey and a topic. I think we're missing the Whisper topic here.